package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "saucelabs-connector",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
We change public API with this PR, so we should change the minor version (`0.1.0`)
FYI: `version: {MAJOR}.{MINOR}.{PATCH}` (http://semver.org/)